src/sensors/gps_display.h

```diff
@@ -43,6 +42,8 @@ class MapWidget;
 
 /**
  * Retrieves the GPS position and displays a marker at this position on a MapWidget.
+ * 
+ * \todo Use qreal instead of bool (affects all sensor code)
```

Comment:
Oops. `Use qreal instead of *float* (affects all sensor code)`

We get `qreal`  from Qt, we pass `qreal` to Qt, but internally we use `float` and do a lot of conversation for no (or little) benefit. This is a task for `dev`. I even created a Github project and card on this occassion.

Reply:
Ack.